instrumentation/instaawssdk/instaawssdk.go

```diff
@@ -96,5 +108,44 @@ func injectTraceContext(sp opentracing.Span, req *request.Request) {
 			opentracing.TextMap,
 			SNSMessageAttributesCarrier(params.MessageAttributes),
 		)
+	case *lambda.InvokeInput:
+		lc := LambdaClientContext{}
+
+		if params.ClientContext != nil {
+			res, err := base64.StdEncoding.DecodeString(*params.ClientContext)
+			if err != nil {
+				sp.LogFields(otlog.Error(req.Error))
+
+				return
+			}
+
+			err = json.Unmarshal(res, &lc)
+			if err != nil {
+				sp.LogFields(otlog.Error(req.Error))
+
+				return
+			}
+		}
+
+		if lc.Custom == nil {
+			lc.Custom = make(map[string]string)
+		}
+
+		sp.Tracer().Inject(
+			sp.Context(),
+			opentracing.TextMap,
+			opentracing.TextMapCarrier(lc.Custom),
+		)
+
+		s, err := lc.Base64Json()
+		if err != nil {
+			sp.LogFields(otlog.Error(req.Error))
+
+			return
+		}
+
+		if len(s) <= maxClientContextLen {
+			params.ClientContext = &s
```

Comment:
I believe AWS SDK has `aws.String()` for this.

Reply:
I think in this case it is doing the same thing. 

```
func String(v string) *string {
	return &v
}
```